src/lib/orm/orm.go

```diff
@@ -84,27 +88,34 @@ func Clone(ctx context.Context) context.Context {
 // WithTransaction a decorator which make f run in transaction
 func WithTransaction(f func(ctx context.Context) error) func(ctx context.Context) error {
 	return func(ctx context.Context) error {
+		_, span := tracelib.StartTrace(ctx, tracerName, "start-transaction")
```

Comment:
It is better to pass the operation name by context and display the operation name instead of "start-transaction"

Reply:
we need to check the nil of span that returned by StartTrace. Or we can do it in the StartTrace generally.

Reply:
nil span will create a new span from scratch